src/main/java/teammates/common/datatransfer/StudentAttributesFactory.java

```diff
@@ -8,107 +8,104 @@
+     * normal enroll line. <br>
+     * Pre-condition: headerRow must not be null
+     * 
      * @throws EnrollException
      */
-    public StudentAttributesFactory(String headerRow) throws EnrollException {    
+    public StudentAttributesFactory(String headerRow) throws EnrollException {
         Assumption.assertNotNull(headerRow);
-        
+
         int fieldCount = locateColumnIndexes(headerRow);
-        
+
         if (fieldCount < MIN_FIELD_COUNT || !hasTeam || !hasName || !hasEmail) {
-            
             String missingField = "";
-            missingField = hasTeam? missingField : missingField + " <mark>Team</mark>";
-            missingField = hasName? missingField : missingField + " <mark>Name</mark>";
-            missingField = hasEmail? missingField : missingField + " <mark>Email</mark>";
+            missingField = hasTeam ? missingField : missingField + " <mark>Team</mark>";
+            missingField = hasName ? missingField : missingField + " <mark>Name</mark>";
+            missingField = hasEmail ? missingField : missingField + " <mark>Email</mark>";
             throw new EnrollException(ERROR_HEADER_ROW_FIELD_MISSED + ": " + missingField);
-        } 
+        }
     }
-    
+
     /**
      * Return a StudentAttributes object created from the given enrollLine.
+     * 
      * @throws EnrollException
      */
     public StudentAttributes makeStudent(String enrollLine, String courseId)
             throws EnrollException {
-        
         if (enrollLine.isEmpty()) {
             throw new EnrollException(ERROR_ENROLL_LINE_EMPTY);
         }
 
         String[] columns = splitLineIntoColumns(enrollLine);
 
-        boolean hasMissingFields = columns.length <= teamColumnIndex 
-                                || columns.length <= nameColumnIndex 
+        boolean hasMissingFields = columns.length <= teamColumnIndex
+                                || columns.length <= nameColumnIndex
                                 || columns.length <= emailColumnIndex;
         if (hasMissingFields) {
             throw new EnrollException(ERROR_ENROLL_LINE_TOOFEWPARTS);
         }
-        
+
         String paramTeam = columns[teamColumnIndex].trim();
         String paramName = columns[nameColumnIndex].trim();
         String paramEmail = columns[emailColumnIndex].trim();
-        
-        String paramComment;
+
+        String paramComment = "";
         if (hasComment && columns.length > commentColumnIndex) {
             paramComment = columns[commentColumnIndex].trim();
-        } else {
-            paramComment = "";
         }
 
-        String paramSection;
-        if(hasSection && columns.length > sectionColumnIndex){
+        String paramSection = Const.DEFAULT_SECTION;
+        if (hasSection && columns.length > sectionColumnIndex) {
             paramSection = columns[sectionColumnIndex].trim();
-        } else {
-            paramSection = Const.DEFAULT_SECTION;
         }
```

Comment:
The original was more intuitive/natural? I guess it is subjective. The original reads "if the condition is met, set the section to x, otherwise set it to default" while the new code reads "set the section to default, if the condition is set, set it to x".